packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/AccessRequirementWikiInstructions.test.tsx

```diff
@@ -7,37 +6,22 @@ import {
   mockToUAccessRequirementWithWiki,
 } from '../../mocks/mockAccessRequirements'
 import { mockToUAccessRequirementWikiPage } from '../../mocks/mockWiki'
-import { rest, server } from '../../mocks/msw/server'
+import { server } from '../../mocks/msw/server'
 import SynapseClient from '../../synapse-client'
 import {
   confirmMarkdownSynapseTextContent,
   expectMarkdownSynapseNotToGetWiki,
   waitForMarkdownSynapseToGetWiki,
 } from '../../testutils/MarkdownSynapseUtils'
 import { createWrapper } from '../../testutils/TestingLibraryUtils'
-import { WIKI_PAGE_ID } from '../../utils/APIConstants'
-import { BackendDestinationEnum, getEndpoint } from '../../utils/functions'
 import { NO_WIKI_CONTENT } from '../Markdown/MarkdownSynapse'
 import {
   AccessRequirementWikiInstructions,
   AccessRequirementWikiInstructionsProps,
 } from './AccessRequirementWikiInstructions'
 
 const createWikiPageSpy = jest.spyOn(SynapseClient, 'createWikiPage')
-const changeGetWikiPageHandlerOnce = (wikiPage: WikiPage) => {
```

Comment:
This test failed because it relied on hot-swapping the mock server implementation between saving/updating. This is really hard to get to work reliably, so I removed the hot-swapping and updated the mock server to use an in-memory database (we have done this for a few other mock services, such as teams & challenges).

Now, updates are properly handled by the mock service and seen in the UI.